modules/management/kubectl-apply/main.tf

```diff
@@ -48,3 +53,38 @@ module "kubectl" {
     http    = http.h
   }
 }
+
+module "install_kueue" {
+  source            = "./kubectl"
+  source_path       = local.install_kueue ? local.kueue_install_source : null
+  server_side_apply = true
+
```

Comment:
nit: maybe we explicitly set `wait_for_rollout = true`? to keep it safe from any changes on default values in kubectl module.